glean-core/android/src/main/java/mozilla/telemetry/glean/private/DatetimeMetricType.kt

```diff
@@ -40,45 +72,40 @@ class DatetimeMetricType(
      * @param value The [Date] value to set. If not provided, will record the current time.
      */
     fun set(value: Date = Date()) {
-        /*if (!shouldRecord(logger)) {
-            return
-        }
-
-        @Suppress("EXPERIMENTAL_API_USAGE")
-        Dispatchers.API.launch {
-            // Delegate storing the datetime to the storage engine.
-            DatetimesStorageEngine.set(
-                this@DatetimeMetricType,
-                value
-            )
-        }*/
-        Log.e(LOG_TAG, "DatetimeMetricType.set is a stub")
+        val cal = Calendar.getInstance()
```

Comment:
I didn't pay close attention to this previously, but do now:

Given that most of the time this gets called without a value set, we just use the current date.
I wonder if we should move this "some date or use now" into Rust? This doesn't have to happen right now, but we should keep it in mind.
(we would still need to support passing in the `Date` anyway)

Reply:
This is a very good point. I filed [bug 1556677](https://bugzilla.mozilla.org/show_bug.cgi?id=1556677) to track this.